frontend/src/metabase/query_builder/components/template_tags/TagEditorParam.tsx

```diff
@@ -1,15 +1,12 @@
-import { Component } from "react";
+import { Component, useMemo } from "react";
```

Comment:
The labels for `Variable type` and `Filter widget type` inputs are at a different distance from the input than `Variable name`, `Field to map to` and `Filter widget label` inputs (notice the white area between green area and the input in the screenshot):

![image](https://github.com/metabase/metabase/assets/6830683/ffc4dd2a-d005-42a0-9247-0dd568353da7)

Reply:
This is a good catch, thanks!